BlobStorage/Advanced.cs

```diff
@@ -111,89 +100,80 @@ public static async Task CallBlobAdvancedSamples()
-            PrintServiceClientProperties(blobClient);
+            PrintServiceClientProperties(blobServiceClient);
 
             // Configure storage analytics (metrics and logging) on Blob storage.
-            await ConfigureBlobAnalyticsAsync(blobClient);
+            await ConfigureBlobAnalyticsAsync(blobServiceClient);
 
             // List all containers in the storage account.
-            ListAllContainers(blobClient, "sample-");
+            ListAllContainers(blobServiceClient, "sample-");
 
             // List containers beginning with the specified prefix.
-            await ListContainersWithPrefixAsync(blobClient, "sample-");
+            await ListContainersWithPrefixAsync(blobServiceClient, "sample-");
         }
 
         /// <summary>
         /// Calls samples that demonstrate how to work with a blob container.
         /// </summary>
-        /// <param name="container">A CloudBlobContainer object.</param>
+        /// <param name="container">A BlobContainerClient object.</param>
         /// <returns>A Task object.</returns>
-        private static async Task CallContainerSamples(CloudBlobContainer container)
+        private static async Task CallContainerSamplesAsync(BlobContainerClient container)
         {
-            // Fetch container attributes in order to populate the container's properties and metadata.
-            await container.FetchAttributesAsync();
 
             // Read container metadata and properties.
-            PrintContainerPropertiesAndMetadata(container);
+            await PrintContainerPropertiesAndMetadataAsync(container);
 
             // Add container metadata.
             await AddContainerMetadataAsync(container);
 
-            // Fetch the container's attributes again, and read container metadata to see the new additions.
-            await container.FetchAttributesAsync();
-            PrintContainerPropertiesAndMetadata(container);
+            // read container metadata to see the new additions.
+            await PrintContainerPropertiesAndMetadataAsync(container);
 
             // Make the container available for public access.
             // With Container-level permissions, container and blob data can be read via anonymous request. 
             // Clients can enumerate blobs within the container via anonymous request, but cannot enumerate containers.
-            await SetAnonymousAccessLevelAsync(container, BlobContainerPublicAccessType.Container);
+            await SetAnonymousAccessLevelAsync(container, PublicAccessType.BlobContainer);
 
             // Try an anonymous operation to read container properties and metadata.
             Uri containerUri = container.Uri;
 
             // Note that we obtain the container reference using only the URI. No account credentials are used.
-            CloudBlobContainer publicContainer = new CloudBlobContainer(containerUri);
+            var publicContainer = new BlobContainerClient(containerUri);
```

Comment:
Is this line needed? Looks like `publicContainer` is never used.

Reply:
Removed.